src/types.ts

```diff
@@ -111,16 +143,19 @@ export enum PlaceBidStatus {
 export interface NewBidParameters {
   tokenId: string;
   bidAmount: string; // in wei
+  bidToken: string;
```

Comment:
This is the object the user is going to fill out. We don't want them to be filling out `bidToken`, we're going to be pulling that from the contract for the NFT they are bidding on.